packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -221,6 +221,20 @@ export interface QueueProcessingServiceBaseProps {
    * @default - false
    */
   readonly enableExecuteCommand?: boolean;
+
+  /**
+   * Whether CPU based scaling of container should be enabled should be enabled.
+   *
+   * @default - false
```

Comment:
```suggestion
   * @default false
```

Reply:
if we don't follow alternative prop design (a below comment), we need to make this true so we don't break people. If we default this to false we will need to add a feature flag to this

Reply:
this has been removed under alt prop design